jax/experimental/jet.py

```diff
@@ -32,6 +67,51 @@
 from jax import lax
 
 def jet(fun, primals, series):
+  r"""Taylor-mode higher-order automatic differentiation.
+
+  Args:
+    fun: Function to be differentiated. Its arguments should be arrays, scalars,
+      or standard Python containers of arrays or scalars. It should return an
+      array, scalar, or standard Python container of arrays or scalars.
+    primals: The primal values at which the Taylor approximation of ``fun`` should be
+      evaluated. Should be either a tuple or a list of arguments,
+      and its length should be equal to the number of positional parameters of
+      ``fun``.
+    series: Higher order Taylor-series-coefficients.
+      Together, `primals` and `series` make up a truncated Taylor polynomial.
+      Should be either a tuple or a list of tuples or lists,
+      and its length dictates the degree of the truncated Taylor polynomial.
+
```

Comment:
I did not document the exceptions that are raised, because I did not see this kind of doc in many other functions. But I am aware that it would go here, please let me know if I should add it.

Reply:
Not documenting exceptions sounds good to me.